### PR TITLE
fix: show completion modal

### LINKED ISSUE #46361: The Ctrl / Cmd + Enter shortcut doesn't trigger the completion modal in the RWD projects
### Describe the Issue

A learner on Twitter pointed out that using the Ctrl / Cmd + Enter shortcut to run the tests for the RWD projects doesn't trigger the completion modal.

Clicking the "Run the Tests (Ctrl + Enter)" button triggers the modal as expected if all the tests pass. But using the keyboard shortcut just runs the tests, and the shortcut has to be used again to actually submit the code in the editor as a solution and move on to the next challenge.

### Affected Page

All RWD project pages

### Steps to Reproduce

1. Go to one of the RWD projects like [Build a Survey Form](https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form)
2. Enter a valid solution (can use code from the demo repo here for testing: https://github.com/freeCodeCamp/demo-projects/tree/master/apps/survey-form)
3. Use the Ctrl / Cmd + Enter shortcut to run the tests
4. Check the tests pane to see that all the tests have passed, but the completion modal is absent
5. Submit the project code
    - Use the Ctrl / Cmd + Enter shortcut again
    - Or click the "Run the Tests (Ctrl + Enter)" button to trigger the completion modal, then click the "Submit and go to next challenge (Ctrl + Enter)" button, or use the Ctrl / Cmd + Enter shortcut

### Expected behavior

The Ctrl / Cmd + Enter shortcut should trigger the completion modal like clicking the "Run the Tests (Ctrl + Enter)" button does.

### System

- Device: Laptop
- OS: Linux Mint 20.3
- Browser: Chrome (102.0.5005.61), Firefox (101.0)

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- fix: make ctrl/cmd + enter trigger modal
- test: check modal appears
- refactor: ignore additional solutions

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/46361

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

File: cypress/integration/learn/challenges/projects.js [hunk 1]
```diff
@@ -136,4 +136,53 @@ describe('project submission', () => {
...
+        const { block, superBlock, dashedName, solutions } = challenges.find(
+          ({ title }) => title === projectTitle
+        );
+
+        const url = `/learn/${superBlock}/${block}/${dashedName}`;
+        cy.visit(url);
+
+        solutions[0].forEach(({ contents, fileKey }) => {
+          const tabSelector = `[data-cy=editor-tab-${fileKey}]`;
+          console.log(tabSelector);
```
Comment (ojeytonwilliams) [inline cypress/integration/learn/challenges/projects.js:161]:
```suggestion
```

**Review by moT01 — COMMENTED** [review]
So I think there's still a bug here. I can use cmd+enter while the editor is highlighted and the tests run and the modal shows up. But if I use cmd+enter while the instructions are focused, I still see the old behavior - tests pass, no modal.

If that bug is legitimate, we should probably just try to fix it here.

**Comment by ojeytonwilliams** [comment]
> If that bug is legitimate, we should probably just try to fix it here.

Seems likely, I didn't try and fix the hotkeys.  I'll take a look now.

**Review by ghost — COMMENTED** [review]
Looks Good!!!

**Review by raisedadead — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**